exercises/practice/decimal/.meta/example.rs

```diff
@@ -102,7 +102,9 @@ macro_rules! auto_impl_decimal_ops {
             fn $func_name(mut self, mut rhs: Self) -> Self {
                 Decimal::equalize_precision(&mut self, &mut rhs);
                 Decimal::new(
+                    #[allow(clippy::redundant_closure_call)]
                     $digits_operation(self.digits, rhs.digits),
```

Comment:
The macro pastes the closure here, which is immediately called. clippy complains not to create closures and immediately call them.